lib/modules/apostrophe-global/index.js

```diff
@@ -139,25 +140,55 @@ module.exports = {
       self.expressMiddleware = self.addGlobalToData;
     };
 
-    // Standard middleware. Fetch the global doc and add it to `req.data` as `req.data.global`.
+    // Fetch the global doc and add it to `req.data` as `req.data.global`, if it
+    // is not already present. If it is not already present, skip the
```

Comment:
Typo (I think): "If it is not already present" should be "If it is already present."